routes/utils.js

```diff
@@ -6,6 +6,13 @@ const isNumeric = (value, errorMessage) => {
   }
 };
 
+const isNum = (value) => {
```

Comment:
why do we need this function when we already have isNumeric? I think you can just loop through the array input and call isNumeric on each of them